### PR TITLE
Add specification of Read Ups

### PR DESCRIPTION
**Pull request by GordonDiggs**
@codeclimate/review @mrb 

Outlines how engines should emit Read Ups.

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ An `issue` represents a single instance of a real or potential code problem, det
   "description": "Unused local variable `foo`",
   "categories": ["Complexity"],
   "location": Location,
-  "remediation_points": 500
+  "remediation_points": 500,
+  "content": Read Up
```
Comment (brynary) [inline SPEC.md]: Let's call this "Content", not "Read Up" for the spec

File: SPEC.md [hunk 2]
```diff
@@ -147,6 +148,16 @@ line of the file.
 
 Offsets, however are 0-based. A Position of `{ "offset": 4 }` represents the _fifth_ character in the file. Importantly, the `offset` is from the beginning of the file, not the beginning of a line. Newline characters (and all characters) count when computing an offset.
 
+### Read Ups
+
+Read ups give more information about the issue's check, including a description of the issue, how to fix it, and relevant links. They are expressed as a hash with a `body` key. The value of this key should be a markdown document. For example:
```
Comment (brynary) [inline SPEC.md]: :lipstick: capitalize Markdown

Comment (brynary) [inline SPEC.md]: "Read Up" -> "Content" here also

**Comment by brynary** [comment]
Couple minor things but otherwise LGTM

**Comment by GordonDiggs** [comment]
@brynary updated. Also added a link to the Markdown docs.

**Pull request merged**